bee-hive/tests/bee/workflow.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: beehive/v1alpha1
+kind: Workflow
+metadata:
+  name: beetest-deployment
+  labels:
+    app: testapp
+spec:
+  strategy:
+    type: sequence
+    output: nonal
+  template:
+    metadata:
+      labels:
+        app: testapp
+    agents:
+      - name: beetest
```

Comment:
`agents` lists the agent used in the steps.  So it should be `crewtest`.

Reply:
fixed in both tests